dhall-yaml/tasty/Main.hs

```diff
@@ -32,6 +33,10 @@ testTree =
         , testDhallToYaml
             Dhall.JSON.Yaml.defaultOptions
             "./tasty/data/special"
+        , testDhallToYaml
+            Dhall.JSON.Yaml.defaultOptions
+                { conversion = Dhall.JSON.Conversion "mapKey" "mapValue" }
```

Comment:
I wonder whether we should update these `defaultOptions` to accurately reflect the default CLI options.

(In `testDhallToJSON`, we already use this `Conversion` by default:

https://github.com/dhall-lang/dhall-haskell/blob/6c68f8206df5dce9318a23116a5bd3454b8582cd/dhall-json/tasty/Main.hs#L69-L74

)

Reply:
I have prepared [a patch](https://github.com/dhall-lang/dhall-haskell/commit/3cbf18ff2cc8957d52a9565360ebb5ee49079e62) that adds a `defaultConversion`, that we could apply after this fix.

Reply:
Yeah, it would be a good idea to keep them in sync